ethier/sign.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/divergencetech/ethier/eth"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+)
+
+var signer *eth.Signer
+
+func init() {
+	var signCmd = &cobra.Command{
```

Comment:
`signCmd := &cobra.Command{`

The `:=` is shorthand for `var`, which is only needed for (a) global variables; declaring a variable but not setting an initial value; and (c) when you want to explicitly state the variable's type:

(b)
``` Go
var i int // 0
var j string // <empty string>
var k *cobra.Command // nil
```

(c)
``` Go
i := 0 // implicitly type int
var j int64 = 0 // explicit typing
k := int64(0) // although this is more idiomatic
```